ui/packages/catalog-ui-search/src/main/webapp/component/property/property.collection.view.js

```diff
@@ -97,6 +97,15 @@ define([
                 return _.extend(attributeToVal, childView.toJSON());
             }, {});
         },
+        toPropertyJSON() {
```

Comment:
Is it possible to just use the `toJSON` function for your purposes?

Reply:
I believe the `toPatchJSON` might be what you want then, I'm not sure though.  It only includes attributes that were changed by the user, but I'm not sure if that's what you mean.